run.sh

```diff
@@ -45,17 +45,18 @@ docker run --rm \
   -v ${SCRIPT_DIR}/${API}:/go-sdk/sdk \
   generate-sdk
 
-if [[ "$GITHUB_ACTIONS" = "true" ]]; then
-  # Logicless templates are dumping extra quotes around enum values, so we've
-  # added some padding chars and now need to replace the padding + quotes.
-  # This is preferable to writing an entire huge Java project for a `trim`
-  # function in the template. I hate this. 🫣
-  sed -i -E 's/@@@@"([^"]+)"@@@@/\1/g' ./${API}/*.go
-
-  # Logicless templates are dumping `example:"null"` on every field, so we've
-  # got to remove those.
-  sed -i -E 's/ example:"null"//g' ./${API}/*.go
-else
-  sed -i '' -E 's/@@@@"([^"]+)"@@@@/\1/g' ./${API}/*.go
-  sed -i '' -E 's/ example:"null"//g' ./${API}/*.go
-fi
+# Logicless templates are dumping extra quotes around enum values, so we've
+# added some padding chars and now need to replace the padding + quotes.
+# This is preferable to writing an entire huge Java project for a `trim`
+# function in the template. I hate this. 🫣
+sed -i.bak -E 's/@@@@"([^"]+)"@@@@/\1/g' ./${API}/*.go
```

Comment:
my sed-foo is terrible.  Did you find a work around to the differences between sed in mac and linux?

Reply:
Yeah! Turns out if we just set it and cleanup after, that param just configures extensions to use for local backups.